test/extensions/filters/http/jwt_authn/jwks_async_fetcher_test.cc

```diff
@@ -225,7 +225,8 @@ TEST_P(JwksAsyncFetcherTest, TestNetworkFailureFetchAndRefresh) {
   EXPECT_EQ(out_jwks_array_.size(), 0);
 
   // Expect refresh timer is enabled.
-  EXPECT_CALL(*timer_, enableTimer(expected_duration_, nullptr));
+  const std::chrono::milliseconds expected_refetch_time = std::chrono::seconds(1);
```

Comment:
possible to add a test where this value comes from config?

Reply:
Thanks.  added,  it actually discovered a bug.